app/api/files.v2/database/schemas/relationshipTypes.d.ts

```diff
@@ -0,0 +1,13 @@
+import { ObjectId } from 'mongodb';
```

Comment:
Is the filename correct? These are not relationshipTypes, but rather fileTypes?
Also, as we have discussed, can we remove the 'd' if it was created by hand? I'd vote for meaningful distinction, rather than consistency.